src/coap_io.c

```diff
@@ -187,12 +187,10 @@ coap_socket_bind_udp(coap_socket_t *sock,
          "coap_socket_bind_udp: ioctl FIONBIO: %s\n", coap_socket_strerror());
   }
 
-#ifndef RIOT_VERSION
```

Comment:
Why is this not needed (I don't know enough about RIOT)?

Reply:
This snippet is not compiled if RIOT is off anyway due to the `#ifndef RIOT_VERSION` on Line 180.